### PR TITLE
Add Cgrid capability

### PR DESCRIPTION
**Pull request by DeniseWorthen**
Issue #49 : Update CICE to allow C-grid option for dynamics; see Issue https://github.com/CICE-Consortium/CICE/issues/721 for description of testing done for this change. 

This PR will change baselines even when the C-grid option is not exercised because of the change in ``cicecore/cicedynB/dynamics/ice_dyn_evp.F90``:

```
      call grid_average_X2Y('S', uocn     , grid_ocn_dynu, uocnU   , 'U')
      call grid_average_X2Y('S', vocn     , grid_ocn_dynv, vocnU   , 'U')
      call grid_average_X2Y('S', ss_tltx  , grid_ocn_dynu, ss_tltxU, 'U')
      call grid_average_X2Y('S', ss_tlty  , grid_ocn_dynv, ss_tltyU, 'U')
```

These lines determine whether the variables are averaged as though they were fluxes ("F") vs state variables ("S"). The option "S"  is correct for these terms (ocean velocities and surface tilts). The pre-Cgrid code (ie, current emc/develop) is equivalent to averaging the terms as if they were fluxes ("F").

A test was carried out on cheyenne.intel by retaining the value "F" in these four lines. All UWM Baselines were B4B. The change in baselines will result solely from making the change from F->S.

### CODE REVIEW

File: configuration/scripts/options/set_nml.dynanderson [hunk 1]
```diff
@@ -1,3 +1,5 @@
 kdyn            = 3
 algo_nonlin     = 'anderson'
 use_mean_vrel   = .false.
+capping_method  = 'max'
+
```
Comment (BrianCurtis-NOAA) [inline configuration/scripts/options/set_nml.dynanderson:5]: Just wanted to double check that nothing will complain about the extra line here?

Comment (DeniseWorthen) [reply configuration/scripts/options/set_nml.dynanderson:5]: This is used for the standalone testing  so it should have no impact.

**Review by binli2337 — APPROVED** [review]

**Review by BrianCurtis-NOAA — APPROVED** [review]
Approved, but curious if the two empty files are being kept for a reason?

**Comment by DeniseWorthen** [comment]
@BrianCurtis-NOAA What two empty files are you referring to?
 
EDIT: Ok, I see them here on the PR page but they're not empty in my actual branch. I'm not sure why they're showing as empty files.

**Comment by BrianCurtis-NOAA** [comment]
> @BrianCurtis-NOAA What two empty files are you referring to?
> 
> EDIT: Ok, I see them here on the PR page but they're not empty in my actual branch. I'm not sure why they're showing as empty files.

Hmm, it looks like the permissions changed on the file, but maybe the diff is empty and not the file. Thanks!

**Pull request merged**